README.md

```diff
@@ -35,6 +35,8 @@ Test with `hatch env run -e test cov`
 
 > **Note:** tree_maker requires trees be generated from [Clustal](http://www.clustal.org/)!
 
+> **IMPORTANT:** You will need `libgl1` to use this, `apt install libgl1` for Debian based systems.
```

Comment:
I think it is only required for old versions of Ubuntu.
On 22.04 `ligbl` installed with PyQt5 through pip.

For the old version I also needed to install these:
https://github.com/cfe-lab/make_tree/blob/main/.github/workflows/main.yml#L18
(libgl is first of them)